ios/Flutter/flutter_export_environment.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=/Users/barrettbreshears/dev/flutter"
-export "FLUTTER_APPLICATION_PATH=/Users/barrettbreshears/Development/Fediverse"
+export "FLUTTER_ROOT=/home/jff_dev/Apps/flutter"
```

Comment:
Might need to add this to the git ignore so we don't track changes

Reply:
@barrettbreshears Added to .gitignore & removed from tracking